standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/leader/CompactorTasks.java

```diff
@@ -110,6 +113,9 @@ private void logCompactionParameters() {
       if (!MetastoreConf.getBoolVar(configuration, MetastoreConf.ConfVars.COMPACTOR_INITIATOR_ON)) {
         HiveMetaStore.LOG.warn("Compactor Initiator is turned Off. Automatic compaction will not be triggered.");
       }
+      HiveMetaStore.LOG.info(
```

Comment:
As we are already printing the of COMPACTOR_CLEANER_ON as INFO log on line 96, do we really need this log?
Can this be similar to the WARN log for initiator from line 113 and give a warn log if cleaner is turned off and automatic cleaner will be be triggered?

Reply:
Made changes as recommended